core/address.go

```diff
@@ -40,6 +40,7 @@ func IsSmartContractAddress(rcvAddress []byte) bool {
 		return true
 	}
 
+	//TODO: check how a normal address with 8 leading zeroes influences everything
```

Comment:
it doesn't. you can't send a transaction to it.

Reply:
ok